src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -222,7 +221,7 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.InstanceId == "" {
-		cfg.InstanceId = hex.EncodeToString(uuid.NodeID())
+		cfg.InstanceId = uuid.New()
```

Comment:
Yeah exactly -- that sounds exactly like what I've seen too, both with that last pass with this new ID and with our damn emails.